listenbrainz/webserver/static/css/brainzplayer.less

```
@@ -239,9 +247,21 @@
 			border-right: 2px solid @dark-color;
 		}
 		&:hover{
+			transition: height, 0.3s;
 			height: @progress-bar-hover-height;
 			top: -@progress-bar-height - (@progress-bar-hover-height - @progress-bar-height)/2
```

After testing locally, I think now that the progress bar is bigger it works best if it expands only towards the top

```suggestion
			top: -@progress-bar-hover-height;
```

---

Done in upcoming set of changes.